LICENSE.md

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2017
```

Comment:
Yep, just a bootstrap `ember addon` with a few package adjustments - I'll roll in the rest as I start moving code in